### PR TITLE
Added laser shifts to input for each libE config

### PR DESCRIPTION
**Pull request by LDAmorim**
Hi all,
This small PR addresses issue #1140.

Changes:
*  ``Tools/LibEnsemble/write_sim_input.py`` - to include the antenna position and time to reach the peak field for the 2nd laser as a function of the starting position of the 2nd stage (which is scanned as an input parameter of the libEnsemble runs).

* `` Tools/LibEnsemble/sim/inputs`` - to remove extra spaces for query to work in ``write_sim_input.py``

Cheers,
Diana

### CODE REVIEW

File: Tools/LibEnsemble/write_sim_input.py [hunk 1]
```diff
@@ -92,6 +97,13 @@ def write_sim_input(input_file, x_values):
     output_text = _set_value(
         output_text, 'warpx.zmax_plasma_to_compute_max_step = ',
         str(zmax_stop_run))
+    # Set laser2 position
+    output_text = _set_value(
+        output_text, 'laser2.position     = ',
```
Comment (MaxThevenet) [inline Tools/LibEnsemble/write_sim_input.py]:
```suggestion
        output_text, 'laser2.position = ',
```
and then modify `sim/inputs` accordingly, to stop filling these lines with whitespaces.

**Review by MaxThevenet — APPROVED** [review]
Great, thanks for fixing this!
This looks good to me (just see minor comment below), but let's wait for the Summit results indeed.

File: Tools/LibEnsemble/write_sim_input.py (hunk 1 above)
Comment (LDAmorim) [reply Tools/LibEnsemble/write_sim_input.py]: Thank you very much @MaxThevenet for the super quick feedback!
That does make much more sense, yes, thank you.

**Review by MaxThevenet — APPROVED** [review]

**Comment by oshapoval** [comment]
Thanks for the PR, Diana! Fine with me!

**Comment by LDAmorim** [comment]
Thank you Olga!!

**Comment by LDAmorim** [comment]
This PR has been successfully tested at Summit - OLCF.

File: Tools/LibEnsemble/sim/inputs [hunk 2]
```diff
@@ -195,25 +195,25 @@ warpx.mirror_z_npoints = 4 4
 #################################
 ############# LASER #############
 #################################
-lasers.nlasers     = 2
-lasers.names        = laser1 laser2
+lasers.nlasers = 2
```
Comment (ax3l) [inline Tools/LibEnsemble/sim/inputs:198]: this file contains no changes but white spaces (some of them are now less readable). do we want to change it or reset the changes in it?

Comment (LDAmorim) [reply Tools/LibEnsemble/sim/inputs:198]: Hi @ax3l !
These changes were triggered by:
https://github.com/ECP-WarpX/WarpX/pull/1141#discussion_r449362150.
The way we replace this parameters in LibEnsemble depends on the spaces, that is why it is important to correct them.
So we want to keep the spaces changes.

**Review by ax3l — APPROVED** [review]
Thank you! :sparkles:

**Pull request merged**